app/assets/stylesheets/home.scss

```diff
@@ -6,7 +6,14 @@
   font-size: 22px;
   margin-top: 30px;
 }
+
 .navigation-links {
-  font-size:22px;
-  list-style:none;
+  font-size: 22px;
+  list-style: none;
 }
+
+.field_with_errors {
+  padding: 2px;
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.